develop/tutorials/articles/193-adaptive-media/00-using-adaptive-media-apis-intro.markdown

```diff
@@ -0,0 +1,283 @@
+adaptive media. This tutorial demonstrates these features.
+
+<!-- Are there any other things you can do with the API that we should mention? -->
+
+First, you must deploy the proper Adaptive Media OSGi modules. This is covered
+next.
+
+## Deploying Adaptive Media
+
+Development of Adaptive Media modules is done in the master branch of an
+independent repository of Liferay at the following URL:
+[https://github.com/liferay/com-liferay-adaptive-media](https://www.google.com/url?q=https://github.com/liferay/com-liferay-adaptive-media&sa=D&ust=1482880889869000&usg=AFQjCNHzWR-uBL7ROpGnHzG2agFqs89miQ)
+
+<!-- Shouldn't we just point to the Liferay-portal repo and consider that what
+is in Portal is always stable? I know at the moment the repo is up to date in 
+Portal. -->
+
+<!--
+Periodically this repository will be synchronized with the official
+liferay-portal repository that will contain the latest stable version of
+Adaptive Media in the following url:
+[https://github.com/liferay/liferay-portal/tree/master/modules/apps/adaptive-media](https://www.google.com/url?q=https://github.com/liferay/liferay-portal/tree/master/modules/apps/adaptive-media&sa=D&ust=1482880889870000&usg=AFQjCNHfNqLv5_JXqMiTwLavewkaSg53VQ)
+
+If you want to test the latest features of Adaptive Media, even if they
+are not “official” yet, you should deploy the independent
+`com-liferay-adaptive-media repository`. This contains all the latest features, 
+but it might contain some unfinished or not stable features.
+
+If you want to test the latest official Adaptive Media, you should use
+the one in the liferay-portal repository. Keep in mind that this
+repository will be synced from time to time, so latest features might not
+be available in this repository. -->
+
+To test the Adaptive Media features, you must deploy all the modules contained 
+in the Adaptive Media repo, except the testing ones 
+(the ones with the `-test` suffix).
+
+<!-- Note: there is currently an issue with the `adaptive-media-image-jax-rs` 
+module that causes images to break. -->
+
+Once the Adaptive Media modules are deployed, you can interact with the APIs.
+
+## Adaptive Media API
+
+Media content (images, audio, video…) is represented with the 
+[`com.liferay.adaptive.media.AdaptiveMedia` interface](https://github.com/liferay/com-liferay-adaptive-media/blob/6356d57a76050764d52a8dccce07578cc53b72b1/adaptive-media-api/src/main/java/com/liferay/adaptive/media/AdaptiveMedia.java). 
+This interface exposes the following data about a media:
+
+-   A set of attributes (content length, content type, dimensions,
+    etc.).
+-   The raw content of the media.
+-   A URI that uniquely identifies the media.
+
+The set of attributes allowed for a media depend on its type. For example, 
+images have different characteristics from videos or audio files, and the API 
+tries to enforce this statically.
+
+<!-- How do you create a media type? Should we mention that in this tutorial, or 
```

Comment:
I think that we should focus on Adaptive Media images for now. In the future we might expand it, but let's focus on images for now.